docs/cli.rst

```diff
@@ -1,11 +1,248 @@
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Troubleshooting Errors
+----------------------
+
+Errors happen! That’s why our team strives to provide our users with options for efficiently working with them when they occur.
+
+The ``--debug`` Option
+^^^^^^^^^^^^^^^^^^^^^^
+All CumulusCI commands can be passed the ``--debug`` option. When this is used, the following occurs:
+    * Any calls to CumulusCI's logger at the debug level are shown.
+    * Any errors captured by ``requests.packages.urllib3.add_stderr_logger()`` are shown.
+    * If an error is present, the corresponding stacktrace is shown. 
+    * The user is dropped into a `post-mortem debugging <https://docs.python.org/3/library/pdb.html#pdb.post_mortem>`_ session.
+
+Log Files
+^^^^^^^^^
+CumulusCI creates a log file every time a cci command besides ``gist`` is run. There are six rotating logfiles (``cci.log, cci.log1...5``) with ``cci.log`` being the most recent. Logfiles are stored under ``~/.cumulusci/logs``. By default, logfiles capture the the following:
+    * The last command that was entered by the user
+    * All output from the command (including debug information)
+    * If an error is present, the corresponding stacktrace is included.
+
+If you want debug information regarding the ``requests`` module to be captured in a logfile you must explicitly run the command with the ``--debug`` option.
+
+Viewing Stacktraces
+^^^^^^^^^^^^^^^^^^^
+If you encounter an error and want more information on what went wrong, you can use ``cci error info`` to display the last *n* lines of the stacktrace (if present) from the last command you executed in CumulusCI.
+
+By default, CumulusCI displays the last 30 lines from the stacktrace. You can use the ``-m`` or ``--max-lines`` option specify how much of the stacktrace you would like to see. The following displays the last 10 lines from the stacktrace:
+
+.. code-block:: console 
+
+    cci error info --max-lines 10
+
+Reporting Error Logs 
```

Comment:
What are your thoughts about moving this section upwards, and starting with more "regular-user-facing" material before descending to more developer- or CCI-developer-oriented options?